rmw/include/rmw/rmw.h

```diff
@@ -1996,31 +1994,60 @@ rmw_wait(
   rmw_wait_set_t * wait_set,
   const rmw_time_t * wait_timeout);
 
-/// Return a list of node name and namespaces discovered via a node.
+/// Return all known nodes' names and namespaces.
```

Comment:
This is grammatically correct, but hard to read (in my opinion).  Maybe:

```suggestion
/// Return the name and namespace of all known nodes.
```

Reply:
See https://github.com/ros2/rmw/pull/272/commits/cbabbacaa854d3087e67a57788b6eaf1333aed98.